gtars/src/uniwig/counting.rs

```diff
@@ -1319,3 +1464,48 @@ fn set_up_file_output(
         // write to std_out, this will be useful for sending input to bigtools to create bw files
     }
 }
+
+pub fn get_shifted_pos(flags: Flags, start_site:i32, end_site:i32) -> i32 {
```

Comment:
Do we really need to pass `flags` like this? This will take ownership of `flags` and I wonder if borrowing (pass by reference) would be better? (i.e. `&Flags`)

Reply:
I'll try this.

Reply:
This worked and is done.